jenkins/envs/daint_nvcc_cray.sh

```diff
@@ -7,6 +7,7 @@ module switch cudatoolkit/11.2.0_3.39-2.1__gf93aa1c
 module switch cce/10.0.2
 
 export GTCMAKE_GT_CLANG_CUDA_MODE=NVCC-CUDA
+export GTCMAKE_CMAKE_CUDA_FLAGS_RELEASE='-O3 -DNDEBUG --use_fast_math'
```

Comment:
Should we really do fastmath?

Reply:
That’s a leftover from the other branch. Not sure if it even matters here, will remove it for now. I just saw that it was enabled on CPU backends, so enabled it here, too, to make sure I don’t loose any performance because of that.